### PR TITLE
Created-Body

### PR DESCRIPTION
**Pull request by Uwiclemy**

### CODE REVIEW

File: client/src/components/Body.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+
+class Body extends Component {
+  render() {
+    return (
+      <main className="homePage">
+        <h1 className="pageTitle">Class2 Project</h1>
+        <img
+          className="image"
+          src="//image.shutterstock.com/image-vector/code-bee-logo-design-template-450w-721374241.jpg"
```
Comment (Toinne) [inline client/src/components/Body.jsx]: PP: import it locally

**Review by Toinne — APPROVED** [review]

File: client/src/components/Body.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+
+class Body extends Component {
+  render() {
+    return (
+      <main className="homePage">
+        <h1 className="pageTitle">Class2 Project</h1>
```
Comment (JoeCamacho) [inline client/src/components/Body.jsx]: Hey Clemy :) What do you think about changing this to a h2 tag and add a paragraph
after that to place a text with the project explanation?

File: client/src/components/Body.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        <img
+          className="image"
+          src="//image.shutterstock.com/image-vector/code-bee-logo-design-template-450w-721374241.jpg"
+          alt="code bee logo design template"
+          title="code bee logo design template"
+        />
+        <p className="paragraph">
```
Comment (JoeCamacho) [inline client/src/components/Body.jsx]: I think we could also ass a h2 tag for "About the Team" here followed with the paragraph with its description

**Review by JoeCamacho — DISMISSED** [review]

File: client/src/components/Body.jsx (hunk 1 above)
Comment (Uwiclemy) [reply client/src/components/Body.jsx]: thanks, i am going to do it!

**Comment by JoeCamacho** [comment]
I made the changes in coordination with Clemy

**Pull request merged**